bigbluebutton-html5/imports/ui/components/settings/submenus/closed-captions/component.jsx

```diff
@@ -0,0 +1,259 @@
+                aria-labelledby='ccLanguageLabel' aria-describedby='ccLanguageDesc'
+                onChange={this.selectHandler.bind(this, "ccLocale", "locales")}>
+                <option value='-1' disabled>{this.props.locales && this.props.locales.length > 0 ? "Choose language" : "No active locales" }</option>
+                {this.props.locales ? this.props.locales.map((locale, index) =>
+                  <option key={index} value={index}>{locale}</option>
+                )
+                : null }
+              </select>
+            <div id='ccLanguageLabel' hidden>Language</div>
+            <div id='ccLanguageDesc' hidden>Chooses the language from currently active locales.</div>
+          </div>
+        </div>
+
+        <div className={styles.indentedRow} role='presentation'>
+          <div className={styles.containerLeftHalf}>Font-Family</div>
+          <div className={styles.containerRightHalfContent} role='presentation'>
+              <select defaultValue={this.state.ccFontFamily} tabIndex='10'
+                aria-labelledby='ccFontFamilyLabel' aria-describedby='ccFontfamilyDesc'
+                onChange={this.selectHandler.bind(this, "ccFontFamily", "fonts")}>
+                <option value='-1' disabled>Choose font-family</option>
+                {this.props.fonts ? this.props.fonts.map((font, index) =>
+                  <option key={index} value={index}>{font}</option>
+                )
+                : null }
+              </select>
+            <div id='ccFontFamilyLabel' hidden>Font-Family</div>
+            <div id='ccFontfamilyDesc' hidden>Chooses the Font-Family from the dropdown menu.</div>
+          </div>
+        </div>
+
+        <div className={styles.indentedRow} role='presentation'>
+          <div className={styles.containerLeftHalf}>Font-Size</div>
+          <div className={styles.containerRightHalfContent} role='presentation'>
+              <select defaultValue={this.state.ccFontSize} tabIndex='11'
+                aria-labelledby='ccFontSizeLabel' aria-describedby='ccFontSizeDesc'
+                onChange={this.selectHandler.bind(this, "ccFontSize", "fontSizes")}>
+                <option value='-1' disabled>Choose Font-size</option>
+                {this.props.fontSizes ? this.props.fontSizes.map((size, index) =>
+                  <option key={index} value={index}>{size + 'px'}</option>
+                )
+                : null }
+              </select>
+            <div id='ccFontSizeLabel' hidden>Font-Size</div>
+            <div id='ccFontSizeDesc' hidden>Chooses the Font-Size from the dropdown menu.</div>
+          </div>
+        </div>
+
+        <div className={styles.indentedRow} role='presentation'>
+          <div className={styles.containerLeftHalf}>Background Color</div>
+          <div className={styles.containerRightHalfContent} role='presentation'>
+            <div
+              tabIndex='12'
+              className={ styles.swatch }
+              style={{
```

Comment:
Why is there so many inline styles? couldn't them be inside classes? here for instance on `styles.swatch`?